src/Chapter1.hs

```diff
@@ -490,8 +491,12 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
 
+lastDigit :: Int -> Int
+lastDigit n
+      | n < 0 = mod (negate n) 10
+      | n `mod` 10 == 0 = 0
+      | otherwise = n `mod` 10
```

Comment:
Nice! It could be simplified a bit though :)

This is actually could be combined into just one case, because in case of `n mod 10` equal 0 it still returns 0 which is covered by just returning `n mod 10`.

While doing this simplification, you can notice that it actually could be just 1 case if you use `abs` function on the input 👌🏼